testing/py3-pytest-timeout/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=py3-pytest-timeout
+pkgver=1.3.3
+pkgrel=0
+pkgdesc="py.test plugin to abort hanging tests"
+url="https://bitbucket.org/pytest-dev/pytest-timeout"
+arch="noarch"
+license="MIT"
+depends="py3-pytest"
+makedepends="py3-setuptools"
+checkdepends="py3-pexpect py3-tox"
+subpackages="$pkgname-doc"
+source="https://files.pythonhosted.org/packages/13/48/7a166eaa29c1dca6cc253e3ba5773ff2e4aa4f567c1ea3905808e95ac5c1/pytest-timeout-$pkgver.tar.gz"
+builddir="$srcdir/pytest-timeout-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	tox -e python3.7 -- \
+		--deselect=test_pytest_timeout.py::test_fix_setup[function-meth0] \
```

Comment:
shellcheck warns that the `[]` in here are interpreted as character classes. Perhaps escape/quote them?

Reply:
Done! Thank you!